src/http_server.h

```diff
@@ -367,6 +500,14 @@ class HTTPAPIServer : public HTTPServer {
   re2::RE2 systemsharedmemory_regex_;
   re2::RE2 cudasharedmemory_regex_;
   re2::RE2 trace_regex_;
+
+  // [DLIS-5551] currently always perform basic conversion
+  std::unique_ptr<MappingSchema> generate_request_schema_{new MappingSchema()};
```

Comment:
Question: does basic conversion include schema mapping?

If not could we comment out / error that code for clarity in this version?

Reply:
Is this going to block PR approval? Commenting out this needs to touch many places and soon to be un-commented. It will error out already if user pass a complex message, see `test_complex_schema`.

Reply:
It's more about code readability / reasoning - difficult to understand if its supported or not. Also may decide not to support it / etc.

Maybe just a comment before the convert functions?